### ISSUE TITLE
Apply new time eligibility ru

### ISSUE THREAD

**Issue by NickSchimek** [codeforpdx/recordexpungPDX#387]
## Description
If the most recent conviction is eligible and if there are only other "arrests" on that case then the whole case is time eligible.

The term 'arrests' means the charge was acquitted.

Should wait until #382 is done before working on this.

Please add appropriate test cases for expected behavior.

Please reach out to `@poppa_nick` on slack if any questions.

### This task is done when:
 - arrests in the case belonging to the MRC are marked as time eligible.

**Comment by NickSchimek** [comment]
@aemrich Here is the tests that are needed to check correct behavior is implemented.
```
# These are assuming MRC charge is eligible
1. case = [MRC charge, arrest]                       # Whole case is eligible
2. case = [MRC charge, violation, arrest]            # Whole case is eligible
3. case = [MRC charge, violation, violation, arrest] # Case NOT eligible
4. case = [MRC charge, conviction, arrest]           # Case NOT eligible

# These are assuming MRC charge is NOT eligible
1. case = [MRC charge, arrest]                       # Case is NOT eligible
2. case = [MRC charge, violation, arrest]            # Case is NOT eligible
3. case = [MRC charge, violation, violation, arrest] # Case is NOT eligible
4. case = [MRC charge, conviction, arrest]           # Case is NOT eligible
```

**Comment by wittejm** [comment]
closed with #716
